core/src/main/resources/log4j4ibis.xml

```diff
@@ -90,6 +90,7 @@
 		<Logger name="nl.nn.adapterframework.testtool" level="WARN" />
 		<Logger name="org.apache" level="INFO" />
 		<Logger name="org.apache.commons" level="WARN" />
+		<Logger name="org.apache.http" level="INFO" /><!-- allows for easily changing the log level -->
```

Comment:
Handiger/beter om dit dynamisch te kunnen doen in de /iaf/gui/#!/logging/settings. Dus daar in text box een logger definitie + settings toe kunnen voegen.

Reply:
Created issue https://github.com/ibissource/iaf/issues/3549